core/src/main/scala/me/lyh/protobuf/generic/Schema.scala

```diff
@@ -64,15 +70,32 @@ object Schema {
     case LABEL_REPEATED => Label.REPEATED
   }
 
+  private[generic] def optionMap(options: MessageOrBuilder): Option[Map[String, String]] = {
+    if (options.getAllFields.size > 0) {
+      val optionsMap = options.getAllFields.asScala.foldLeft(Map.empty[String, String]) {
+        (map, field) => {
+          field match {
+            case (desc, ref) => map + (desc.getFullName -> ref.toString)
+          }
+        }
+      }
+      Some(optionsMap)
+    } else None
+  }
 }
 
 private[generic] object SchemaMapper {
 
-  private val schemaMapper = new ObjectMapper().registerModule(DefaultScalaModule)
+  private val schemaMapper = new ObjectMapper()
+    .setSerializationInclusion(JsonInclude.Include.NON_EMPTY)
+    .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
`setSerializationInclusion(JsonInclude.Include.NON_EMPTY)` doesn't serialize empty values
`.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)` - make this version compatible with future changes where we add more stuff to schemas and fields.

Reply:
Note: This will remove empty `enum` field from output result